templates/prelogin/demo.html

```diff
@@ -0,0 +1,126 @@
+{% extends 'prelogin/base.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block stylesheets %}
+    <link href="{% static "app_manager/js/vellum/style.css" %}" type="text/css" rel="stylesheet"/>
+    <style>
+        .form-signup {
+            margin-left: 0;
+        }
+    </style>
+{% endblock stylesheets %}
+
+{% block js-inline %}
+    <script>
+        $(function(){
+            var signUpFormHandler = function() {
+                var submittedForm = $(this);
+                submittedForm.find(".btn[type='submit']").attr('disabled', 'disabled').prepend('<i class="fa fa-refresh fa-spin icon-refresh icon-spin"></i>');
+                kmqPushSafe(['record', 'Clicked "Sign up" from demo', {}, function() {
+                    submittedForm.off('submit', signUpFormHandler).submit();
+                }]);
+                return false;
+            };
+            $('form.form-signup').on('submit', signUpFormHandler);
```

Comment:
I'd like to propose thinking about this as a test - meaning, we worry a lot
about what we are going to measure and not that much about getting all the
ux details right. We can leave this up for a week and see if "demo" is even
a thing people will want to click on, how they do in the form builder, etc.
I would definitely want to track the demo in fullstory as well.

As a next step, Jenny, do you want to work on a test document together? We
can borrow the growth team template and put it through that process.

Amelia



On Thu, Aug 4, 2016 at 6:19 PM, Ben Rudolph <notifications@github.com>
wrote:

> In templates/prelogin/demo.html
> <https://github.com/dimagi/commcarehq-prelogin/pull/92#discussion_r73612940>
> :
>
> > +        }
> > +    </style>
> > +{% endblock stylesheets %}
> > +
> > +{% block js-inline %}
> > +    <script>
> > +        $(function(){
> > +            var signUpFormHandler = function() {
> > +                var submittedForm = $(this);
> > +                submittedForm.find(".btn[type='submit']").attr('disabled', 'disabled').prepend('<i class="fa fa-refresh fa-spin icon-refresh icon-spin"></i>');
> > +                kmqPushSafe(['record', 'Clicked "Sign up" from demo', {}, function() {
> > +                    submittedForm.off('submit', signUpFormHandler).submit();
> > +                }]);
> > +                return false;
> > +            };
> > +            $('form.form-signup').on('submit', signUpFormHandler);
>
> suppose your version tracks some analytics, but that could still be done
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcarehq-prelogin/pull/92/files/e2c75100b4622f6f2d21998c6705a8982f096251#r73612940>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAazfChySCkc6j80fdIlpAb98Cz7dqtaks5qcmWJgaJpZM4JcJB1>
> .
>